packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -31,8 +31,8 @@ import { GoogleWorkspaceInviteButton } from "./GoogleWorkspaceInviteButton";
 type MemberInvitationModalProps = {
   isOpen: boolean;
   onExit: () => void;
-  onSubmit: (values: NewMemberForm) => void;
-  onSettingsOpen?: () => void;
+  onSubmit: (values: NewMemberForm, resetFields: () => void) => void;
+  onSettingsOpen: () => void;
```

Comment:
I don't think this should be required, as it causes type errors

```suggestion
  onSettingsOpen?: () => void;
```